generators/contract/templates/java/src/main/java/org/example/MyContract.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * <%= spdxAndLicense // SPDX-License-Identifier: Apache-2.0 %>
+ */
+package org.example;
+
+import org.hyperledger.fabric.contract.Context;
+import org.hyperledger.fabric.contract.ContractInterface;
+import org.hyperledger.fabric.contract.annotation.Contract;
+import org.hyperledger.fabric.contract.annotation.Default;
+import org.hyperledger.fabric.contract.annotation.Transaction;
+
+import io.swagger.v3.oas.annotations.info.Contact;
+import io.swagger.v3.oas.annotations.info.Info;
+import io.swagger.v3.oas.annotations.info.License;
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+@Contract(name = "<%= assetPascalCase %>Contract", 
+    info = @Info(title = "<%= assetPascalCase %> contract", 
+                description = "<%= description %>", 
+                version = "<%= version %>", 
+                license = 
+                        @License(name = "<%= spdxAndLicense // SPDX-License-Identifier: Apache-2.0 %>", 
+                                url = ""), 
+                                contact =  @Contact(email = "<%= name %>@example.com", 
+                                                name = "<%= name %>", 
+                                                url = "http://<%= name %>.me")))
+@Default
+public class <%= assetPascalCase %>Contract implements ContractInterface {
+    public  <%= assetPascalCase %>Contract() {
```

Comment:
Again, do we need this empty constructor?

Reply:
From Oracle...

"You don't have to provide any constructors for your class, but you must be careful when doing this. The compiler automatically provides a no-argument, default constructor for any class without constructors. This default constructor will call the no-argument constructor of the superclass. In this situation, the compiler will complain if the superclass doesn't have a no-argument constructor so you must verify that it does. If your class has no explicit superclass, then it has an implicit superclass of Object, which does have a no-argument constructor."

Preference.. really here - the contract isn't a bean...